### DISCUSSION TITLE
Set createOptionForm Modal Title

### DISCUSSION THREAD

**Discussion by aaronmeder** [filamentphp/filament#8537] [Help] [answered]
On my edit form I have a `createOptionForm` on a Select field that allows mo to create a new related record on the spot.
This works well:

![Screenshot 2023-09-16 at 12 56 13](https://github.com/filamentphp/filament/assets/5564163/e2fe7f0c-6538-456e-8051-92ea8802e12b)

I would however like to customize the "Create" title in the modal, to "Create Manufacturer" as example. 
Did not find a way to do so in the docs however.

Can that be achieved?

Thanks for feedback
Aaron

**Comment by leandrocfe** [comment, marked as answer]
```php
->createOptionModalHeading('Create Manufacturer')
```

**Reply by aaronmeder** [reply]
@leandrocfe perfect, thanks a lot!

Btw: looks like that is not in the Filament docs. Is there another good resource to be looking for something like this? Or in Filament's code next time? :) 

✨
![Screenshot 2023-09-16 at 13 13 27](https://github.com/filamentphp/filament/assets/5564163/742f3fb2-b43d-440f-90ff-9dccc70bca0d)

**Reply by leandrocfe** [reply]
You can send a PR to update the docs if you want.
Discord topics, GitHub discussions.. you can also explore the source code as Filament offers classes, methods and attributes to assist us.
